Miniforge3/construct.yaml

```diff
@@ -28,6 +28,8 @@ specs:
 
 {% if name.startswith("Mambaforge") %}
   - mamba 0.22.1
+{% else %}
+  - conda-libmamba-solver 22.3.0
```

Comment:
This will pull `libmambapy` plus its deps so I wonder if just the presence of `mamba` as a command is enough to justify the existence of Mambaforge. Like, if we merge, Miniforge and Mambaforge are almost the same. IIRC, Mambaforge was provided separately because it shipped more deps.